### ISSUE TITLE
Attempt to ban a user fails

### ISSUE THREAD

**Issue by mwanner2** [localwiki/localwiki#774]
An attempt to ban a user results in a message "Looks like something went wrong and we're unable to display the page you requested."

**Comment by philipn** [comment]
Non-global admin can't currently block IP addresses - they can only block usernames, hence the error. I'm leaving this issue open as this should either be made more clear or fixed.
